.circleci/config.yml

```diff
@@ -266,6 +269,7 @@ jobs:
             set +x; source activate esmvaltool; set -x
             # Install
             conda install -y esmvaltool -c esmvalgroup -c conda-forge -c birdhouse
+            Rscript ./esmvaltool/R/setup.R > /logs/R_install.txt 2>&1
```

Comment:
This will not work, because conda installation location is not the current working directory. I think we need to add an entry point `esmvaltool-install-dependencies` in setup.py, that runs a Python function that calls Rscript with the correct file (and later on also the julia install script).